.chloggen/target-allocator-use-recommended-prometheus-operator-interfaces.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: target allocator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Use recommended interfaces(resource selector) by the prometheus-operator for watching CRs.
+
+# One or more tracking issues related to the change
+issues: [2309]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

We need to call out that the target allocator now requires get/list/watch permissions for namespaces.

```suggestion
subtext: The target allocator now requires get/list/watch permissions for namespaces. Update your RBAC permissions for the attached role, if necessary.
```